src/cpu/dyn_cache.h

```diff
@@ -710,12 +723,50 @@ static void cache_block_closing(const uint8_t *block_start, Bitu block_size);
 #endif
 
 /* Define temporary pagesize so the MPROTECT case and the regular case share as much code as possible */
-#if defined(HAVE_MPROTECT)
+#if defined(HAVE_MPROTECT) || defined(HAVE_MMAP)
 #define PAGESIZE_TEMP PAGESIZE
 #else
 #define PAGESIZE_TEMP 4096
 #endif
 
+static constexpr size_t cache_code_start_sz = CACHE_TOTAL + CACHE_MAXSIZE + PAGESIZE_TEMP - 1 + PAGESIZE_TEMP;
```

Comment:
A start that has a "size" feels strange - "I'm 50% through the start".  Is there a name that still describes what this is, but is more amenable to having a size?

Reply:
agree, this was quick hack naming to let me keep track of things early on in the experiment, simply `cache_code_size` makes sense?

Reply:
It does; I'm instantly more informed!

Reply:
Feel free to hit 'resolved' whenever you like.  I often resolve as soon as I've got edits that address a comment, even before I push or test (can always unresolve). 👍